src/guide/security/operational-security.md

```diff
@@ -0,0 +1,100 @@
+
+Following and adopting the guidelines in this topic is a necessary step towards achieving total security, however it is not sufficient on its own. To further improve your security, learn more throughout the rest of the [Security](./nav-security.md) section and specifically the following topics:
+
+- [Security Principles](./security-principles.md)
+- [Password Security](./password-security.md)
+
+## Recommended OPSEC Measures
+
+- Stay vigilant. The [most likely](https://arxiv.org/pdf/2209.08356.pdf) way in which one can lose their assets in a blockchain is by giving away their sensitive details.
+
+- Encrypt your disks. Encrypting boot devices allows them to protect your data even if an attacker have gained access of the hardware. Doing it for your portable devices is twice as important.
+
+- Use trusted software. Software that ships via reproducible binary builds, and that you build from source, is the most trustworthy. Proprietary or open-source software that hasn't been audited is a potential risk that must be taken seriously.
+
+- Never leave portable devices with sensitive data unattended. A split second is enough to steal your device.
+
+- Verify the signatures on binary packages. This is not too different from the public key cryptography used inside Iroha v2.
+
+- To prevent unauthorized access, always secure your laptop or personal computer when leaving it unattended. Use strong passwords, lock the screen, and follow best practices for securing your devices.
+
+- Establish a secure [air-gapped](https://en.wikipedia.org/wiki/Air_gap_(networking)) location for your keys. Frist, encrypt they keys, then store them in an _offline-only_ device, ideally with electromagnetic shielding installed. [Hardware keys](./storing-cryptographic-keys.md#using-a-hardware-key) are specifically designed this purpose.
+
+- Always keep your software updated to their latest version across all devices, including computers and phones. Regular updates help patch vulnerabilities and minimise potential risks associated with outdated software, even before such vulnerabilities are disclosed.
+
+- Develop a routine for periodically updating passwords and cryptographic keys. This proactive approach significantly contributes to enhancing overall security posture, since it is much harder to hit a moving target.
+
+## Using Browsers
+
+If an application connected to Iroha 2 features a web UI, your browser can either aid the security or pose a potential threat. It is essential to exercise caution, especially when it comes to the plugins you choose to install.
+
+Consider the following measures to enhance your browsing security:
+
+- Avoid using browsers with known bad security models. Specifically, the following:
+  - Chrome — proprietary.
```

Comment:
I don't think just saying "proprietary" explains what the issue with this browser is. The issue is that it's not audited, right?

If there are articles/research of potential vulnerabilities/cases of actual data leaks/etc. for the listed browsers, I suggest linking them.

Reply:
> I don't think just saying "proprietary" explains what the issue with this browser is.

"Proprietary" points out a lousy security model for a browser. Web browsers are based on the **open specifications** by W3C making proprietary makes them harder to check, security-wise.

Seeing the code helps audit software a lot, even if some may disagree with me.

> The issue is that it's not audited, right?

It's worse. Let's review the current situation:

* Security audit for a browser is already challenging, given how complex both standards and code are. You have a complicated [browser engine](https://en.wikipedia.org/wiki/Comparison_of_browser_engines) like WebKit (Blink) or Gecko, you have a JIT runtime for JS, and you have plugins. In addition to that, browsers typically add code that has access to your pages. In addition to that, there are extensions to think of.
* In a proprietary system, all of that is hidden through obscurity, and audit is complicated
* In a proprietary system, each subsequent update nullifies the result of the previous audit

> If there are articles/research of potential vulnerabilities/cases of actual data leaks/etc. for the listed browsers, I suggest linking them.

Hmm, let's start with the fact browsers are a good target due to their complexity.
Vulnerabilities for these programs are traded, and I may look through the exploit databases. However, strictly speaking, I doubt the exploit databases related to the company maintaining the browsers would remain neutral.
To point out how monumental the problem is, Mozilla made Rust to address many possible types of such vulnerabilities.
At the same time, things like privacy violations would not be recorded, and here we're talking about your data not being sent to the company making the browser and their partners.

I've looked up info on browser privacy, and the first link I see is [here](https://www.unixsheikh.com/articles/choose-your-browser-carefully.html#librewolf).

I remember Aleksandr found another great article, and I will also ask him.

Generally, I recommend simply picking a company, maintaining a browser and looking up "company_name privacy violations" in your search engine.

Reply:
I rewrote this part a little bit. I don't think we need to go into a lot of details, linking to some place where such issues are discussed for each browser is enough.